src/main/java/uk/gov/pay/adminusers/queue/event/EventMessageHandler.java

```diff
@@ -112,6 +112,10 @@ private void handleDisputeCreatedMessage(Event disputeCreatedEvent) throws JsonP
                     serviceAdmins.stream().map(UserEntity::getEmail).collect(Collectors.toSet()),
                     personalisation
             );
+            logger.info("Processed notification email for disputed transaction",
+                    kv("transaction_id", disputeCreatedEvent.getParentResourceExternalId()),
```

Comment:
Could we call this `payment_external_id` to make clear which transaction? It's probably sensible to also log the dispute id too, not sure whether we log this in other places, but might help.

Reply:
Possibly we could add some stuff onto the MDC so we have more information on all log lines.